bitrise/util.go

```diff
@@ -458,8 +458,14 @@ func removeStepDefaultsAndFillStepOutputs(stepListItem *models.StepListItemModel
 			return err
 		}
 
-		if err := repo.CloneTagOrBranch(stepIDData.IDorURI, stepIDData.Version).Run(); err != nil {
-			return err
+		if stepIDData.Version == "" {
```

Comment:
[in the run_util](https://github.com/bitrise-io/bitrise/pull/796/files#diff-c1f6ac45bd213c8bc540592ec9f39f40d39d4a02f4d15749fdd5e48cbbd96808R784) you used a different structure for a similar issue (selecting a function), I like that solution better, what about using it here too:

```
		var cloneCmd *command.Model
		if stepIDData.Version == "" {
			cloneCmd = repo.Clone(stepIDData.IDorURI)
		} else {
			cloneCmd = repo.CloneTagOrBranch(stepIDData.IDorURI, stepIDData.Version)
		}
		if err := cloneCmd.Run(); err != nil {
			return err
		}
```

Reply:
Yeah, that is a little nicer. I'll refactor.

Reply:
Refactored in https://github.com/bitrise-io/bitrise/pull/796/commits/1116ada7d359b582c2889882a136f36d771686e1